pymc/model/core.py

```diff
@@ -1288,6 +1299,7 @@ def make_obs_var(
         data: np.ndarray,
         dims,
         transform: Any | None,
+        default_transform: Any | None,
```

Comment:
Missing description in the docstrings.

Reply:
Also type hint isn't great, should be `RVTransform | None` (or something like that, don't remember the exact class now)